src/pages/editor/InstitutionSupport.tsx

```diff
@@ -0,0 +1,78 @@
+import LinkIcon from '@mui/icons-material/Link';
+import { LoadingButton } from '@mui/lab';
+import { InputAdornment, TextField, Typography } from '@mui/material';
+import { useMutation } from '@tanstack/react-query';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateCustomerInstitution } from '../../api/customerInstitutionsApi';
+import { PageSpinner } from '../../components/PageSpinner';
+import { setCustomer } from '../../redux/customerReducer';
+import { setNotification } from '../../redux/notificationSlice';
+import { RootState } from '../../redux/store';
+import { CustomerInstitution } from '../../types/customerInstitution.types';
+import { dataTestId } from '../../utils/dataTestIds';
+
+export const InstitutionSupport = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const customerMutation = useMutation({
+    mutationFn: (values: CustomerInstitution) => updateCustomerInstitution(values),
+    onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
+      dispatch(setNotification({ message: t('feedback.success.update_customer'), variant: 'success' }));
+    },
+    onError: () => dispatch(setNotification({ message: t('feedback.error.update_customer'), variant: 'error' })),
+  });
+
+  return (
+    <>
+      {!customer ? (
+        <PageSpinner />
+      ) : (
+        <Formik initialValues={customer} onSubmit={async (values) => customerMutation.mutateAsync(values)}>
+          {({ values, isSubmitting, setFieldValue }: FormikProps<CustomerInstitution>) => (
```

Comment:
husk å ta vekk ting når de tas ut av bruk også :)
```suggestion
          {({ isSubmitting }: FormikProps<CustomerInstitution>) => (
```

Reply:
😬